velox/type/Filter.cpp

```diff
@@ -423,6 +423,18 @@ bool MultiRange::testFloat(float value) const {
   return false;
 }
 
+bool MultiRange::testInt64(int64_t value) const {
```

Comment:
MultiRange filters are not used for integral types. For these types, we only use BigintMultiRange. Hence, MultiRange::testInt64 should just throw.